src/main/java/net/mtrop/doom/tools/decohack/data/DEHActionPointer.java

```diff
@@ -114,6 +114,18 @@
 	MONSTERMELEEATTACK  (-1, "MonsterMeleeAttack", DEHActionPointerType.MBF21, DEHActionPointerParam.SHORT, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT, DEHActionPointerParam.INT ),
 	RADIUSDAMAGE        (-1, "RadiusDamage", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
 	NOISEALERT          (-1, "NoiseAlert", DEHActionPointerType.MBF21 ),
+	HEALCHASE           (-1, "HealChase", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	SEEKTRACER          (-1, "SeekTracer", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	FINDTRACER          (-1, "FindTracer", DEHActionPointerType.MBF21, DEHActionPointerParam.ANGLE_FIXED, DEHActionPointerParam.UINT ),
+	CLEARTRACER         (-1, "ClearTracer", DEHActionPointerType.MBF21 ),
+	JUMPIFHEALTHBELOW   (-1, "JumpIfHealthBelow", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	JUMPIFTARGETINSIGHT (-1, "JumpIfTargetInSight", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.ANGLE_FIXED ),
+	JUMPIFTARGETCLOSER  (-1, "JumpIfTargetCloser", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	JUMPIFTRACERINSIGHT (-1, "JumpIfTracerInSight", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.ANGLE_FIXED ),
+	JUMPIFTRACERCLOSER  (-1, "JumpIfTracerCloser", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	JUMPIFFLAGSSET      (-1, "JumpIfFlagsSet", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	ADDFLAGS            (-1, "AddFlags", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	REMOVEFLAGS         (-1, "RemoveFlags", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
```

Comment:
Probably should have caught this in earlier PRs, But could you just remove the spaces in between the last parameter and the closing parenthesis on lines 95-138? A la:
```
REMOVEFLAGS         (-1, "RemoveFlags", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
```
to:
```
REMOVEFLAGS         (-1, "RemoveFlags", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT),
```

Reply:
Will do! :)